build.rs

```diff
@@ -1,5 +1,25 @@
 extern crate pkg_config;
 
 fn main() {
-    pkg_config::find_library("alsa").unwrap();
+    let missing: String = String::from("\"pkg-config\" \"--libs\" \"--cflags\" \"alsa\"");
+
+    match pkg_config::probe_library("alsa") {
+        Ok(_) => {},
+        Err(e) => {
+            match e {
+                pkg_config::Error::Failure {command, output} => {
+                    if command == missing {
+                        panic! (
```

Comment:
Matching on strings doesn't look very robust to me. E g what if pkg_config decides to swap order of --libs and --cflags?